README.md

```diff
@@ -1 +1,65 @@
-# sky-api-tutorial-public-application-chrome-extension
\ No newline at end of file
+# sky-api-tutorial-public-application-chrome-extension
+
+## Overview
+This tutorial demonstrates the interaction of a client application (Google Chrome extension) with the SKY API authorization service, using the [authorization code flow](https://tools.ietf.org/html/rfc6749#section-1.3.2) with Proof Key for Code Exchange (PKCE) to avoid the need to pass a client secret.
+
+## Installing Locally
+
+### Basic Requirements:
+
+- Familiarity with git commands
+- The latest, stable version of [Git](https://git-scm.com/)
+- The [Chrome Browser](https://www.google.com/chrome/browser/desktop/) installed on your desktop
+- A Google Inbox SDK App ID ([free registration](https://www.inboxsdk.com/register))
+
+### Sky API Requirements:
+
+- **A Blackbaud Developer Subscription Key**
+    - If you have not already done so, be sure to complete the [Getting started guide](https://apidocs.sky.blackbaud.com/docs/getting-started/). This will guide you through the process of registering for a Blackbaud developer account and requesting a subscription to an API product.
+    - Once approved, your subscription will contain a **Primary Key** and a **Secondary Key**.  You can use either key as the subscription key value for the `bb-api-subscription-key` request header when making calls to the API.
+    - You can view your subscription keys on your [Blackbaud Developer Profile](https://developer.sky.blackbaud.com/developer).
+- **A Blackbaud Developer Application ID**
+    - [Register your application](https://developerapp.sky.blackbaud.com/applications) in order to obtain the **Application ID** (client ID).
```

Comment:
Wow. This is super old too... guessing this is copied from an old tutorial :X

Reply:
Link updated